tests/functional/h/views/group_test.py

```diff
@@ -4,33 +4,6 @@
 pytestmark = pytest.mark.usefixtures("init_elasticsearch")
 
 
-class TestGroupCreateController:
-    @pytest.mark.usefixtures("with_logged_in_user")
-    def test_submit_create_group_form_without_xhr_returns_full_html_page(
-        self, filled_out_group_form
-    ):
-        response = filled_out_group_form.submit().follow()
-
-        assert response.text.startswith("<!DOCTYPE html>")
-
-    @pytest.mark.usefixtures("with_logged_in_user")
-    def test_submit_create_group_form_with_xhr_returns_partial_html_snippet(
-        self, filled_out_group_form
-    ):
-        response = filled_out_group_form.submit(xhr=True)
-
-        assert response.body.strip(b"\n").startswith(b"<form")
-        assert response.content_type == "text/plain"
-
-    @pytest.fixture
-    def filled_out_group_form(self, app):
-        response = app.get("/groups/new")
-        group_form = response.forms["deform"]
-        group_form["name"] = "My New Group"
-
-        return group_form
```

Comment:
These are tests for the old group form submission behaviour, which no longer exists: there's no HTML form for creating groups anymore and the endpoint for receiving group-create form submissions no longer exists.

We now have a `GroupCreateController` on the backend, this has unit tests but it hardly does anything: just renders a simple Jinja2 template. Then we have the `CreateGroupForm` component on the frontend, this has unit tests in the normal frontend style. The frontend calls the group-create JSON API endpoint to actually create the group, and this endpoint again has its own tests: both unit tests and functional tests.